process-manager.js

```diff
@@ -11,7 +11,12 @@
 
 const EventEmitter = require('events');
 
-const processManagers = new Map();
+const PROCESS_MANAGERS = new Map();
```

Comment:
When should consts be all caps? I know dependencies shouldn't, but everything else is grey to me